### DISCUSSION TITLE
MAUI Developing for individual platforms

### DISCUSSION THREAD

**Discussion by JHeLiu** [dotnet/maui#24620] [Q&A]
Can MAUI be used to create software only for the Windows platform? I don't want to use Android or iOS, but I've tried and it didn't work out. Please don’t recommend using WPF. I’m wondering if MAUI can be used for this purpose or if future versions will allow selecting individual platforms (Windows, Android, iOS, macOS) when creating a new project so that I can choose to develop only for specific platforms. 

For example, when creating a new project in Visual Studio, I can choose the corresponding platform.

- [x] Windows
- [ ] iOS
- [ ] Android
- [ ] macOS

**Comment by axa88** [comment]
Yes. You can enable and disable in project properties interface, but oddly enough if you want to remove mac you need to do it in the config file directly.

lmgtfy
https://www.google.com/search?q=maui+target+framework

**Reply by JHeLiu** [reply]
I tried this a long time ago, but it couldn't prevent macOS from trying to delete it in the .csproj file. The project can run normally in development, but it cannot be published; it throws an error related to _GenerateAppxPackage.

**Reply by axa88** [reply]
Try it again

**Comment by gautambjain** [comment]
MAUI will use WinUI3 internally for Windows platform.

So you can directly create a WinUI3 project.

**Reply by JHeLiu** [reply]
Yes, you're right. Thank you
